Task2/Kuba/NoughtsAndCrossesWPF/NoughtsAndCrossesWPF/ViewModel/ViewModel.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using System.Windows.Input;
+
+namespace NoughtsAndCrossesWPF
+{
+    public class ViewModel //I'm not sure if I have put correct methods in this class. I feel like it's a bit messed but have no better idea how to organize this.
```

Comment:
methods you put here look fine, the IBoardCommunication methos would also be fine here